app-scientific/sagemath/autobuild/build

```diff
@@ -0,0 +1,8 @@
+export SAGE_LOCAL=/usr/lib/sagemath/local
+make configure
+./configure --disable-editable --prefix=$SAGE_LOCAL --enable-build-as-root
+make
+mkdir -p "$PKGDIR"/usr/lib
+cp -av /usr/lib/sagemath "$PKGDIR"/usr/lib/
```

Comment:
Just use `mv` here, there is no real reason to keep a pre-installed copy in `/usr`.